src/content/docs/en/guides/cms/studiocms.mdx

```diff
@@ -0,0 +1,244 @@
+      export default defineConfig({
+          site: 'https://your-site.com/', // Required for StudioCMS
+          output: 'server',
+          adapter: node({ mode: "standalone" }),
+          integrations: [
+              db(),
+              studioCMS(),
+          ],
+      });
+      ```
+
+  3. (Optional) Create a `studiocms.config.mjs` file to configure StudioCMS:
+
+      ```js title="studiocms.config.mjs"
+      import { defineStudioCMSConfig } from 'studiocms/config';
+
+      export default defineStudioCMSConfig({
+          dbStartPage: true, // You will set this to false in a later step
+      });
+      ```
+</Steps>
+
+Your project should now include these files:
+
+<FileTree title="Project Structure">
+- src/
+- .env
+- astro.config.mjs
+- studiocms.config.mjs (optional if you're not using plugins)
+- package.json
+</FileTree>
+
+## Configuring authentication
+
+StudioCMS requires an encryption key for authentication. You can generate one using OpenSSL:
+
+```bash
+openssl rand --base64 16
+```
+
+Add the key to your `.env` file:
+
+```ini title=".env"
+# Required for StudioCMS authentication
+CMS_ENCRYPTION_KEY="wqR+w...sRcg=="
+```
+
+### Optional: Setting up OAuth providers
+
+StudioCMS supports authentication with GitHub, Discord, Google, and Auth0. To configure:
+
+1. Set up OAuth applications with your chosen providers
+2. Use the callback URL format: `https://your-domain.tld/studiocms_api/auth/<provider>/callback`
+3. Add the credentials to your `.env` file
+
+## Setting up package.json scripts
+
+Update your `package.json` to include the necessary scripts:
+
+```json title="package.json" "--remote"
+{
+    "scripts": {
+        "dev": "astro dev --remote",
+        "build": "astro check & astro build --remote",
+        "astro": "astro"
+    }
+}
+```
+
+## Running your StudioCMS project
+
+<Steps>
+1. Initialize your database schema:
+   <PackageManagerTabs>
+     <Fragment slot="npm">
+     ```bash
+     npx astro db push --remote
+     ```
+     </Fragment>
+     <Fragment slot="pnpm">
+     ```bash
+     pnpm astro db push --remote
+     ```
+     </Fragment>
+     <Fragment slot="yarn">
+     ```bash
+     yarn astro db push --remote
+     ```
+     </Fragment>
+   </PackageManagerTabs>
+
+2. Start your development server:
+   <PackageManagerTabs>
+     <Fragment slot="npm">
+     ```bash
+     npm run dev
+     ```
+     </Fragment>
+     <Fragment slot="pnpm">
+     ```bash
+     pnpm run dev
+     ```
+     </Fragment>
+     <Fragment slot="yarn">
+     ```bash
+     yarn dev
+     ```
+     </Fragment>
+   </PackageManagerTabs>
+
+3. Complete the setup by visiting: `http://localhost:4321/start`
+</Steps>
+
+Your StudioCMS dashboard will be available at `http://localhost:4321/dashboard` in development and `https://your-domain.tld/dashboard` in production.
+
+## Adding a frontend to your StudioCMS project
+
+StudioCMS is headless, so you'll need to build your own frontend or use a plugin. To add the blog plugin:
```

Comment:
I did kind of laugh at what sounds like another step, like,  "Step 8: build your own front end!"

What about something like...

```suggestion
StudioCMS provides a [blog plugin](https://docs.studiocms.dev/en/package-catalog/studiocms-plugins/studiocms-blog/) that will create a blog site for you and allow you to create, edit, and delete blog posts from the StudioCMS dashboard.

To get started with a Studio CMS blog, first install the plugin:
```